examples/customer-credit-rating/deploy/quarkus-native-build-task.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: quarkus-native-build
+spec:
+  params:
+    - name: SUBDIRECTORY
+      type: string
+    - name: DOCKERFILE
+      type: string
+    - name: IMAGE
+      type: string
+  steps:
+    - image: bash:latest
+      name: organize-folders
+      script: |
+        #!/usr/bin/env bash
+        root=$(echo "$(params.SUBDIRECTORY)" | cut -d/ -f1)
+        if [ -d "src" ]
+        then
+          rm -rf src pom.xml target
+        fi
+        mv -f $(params.SUBDIRECTORY)/$(params.SUBDIRECTORY)/* .
+        rm -rf $root
+        if [ ! -d "/home/kogito/.m2/repository" ]
+        then
+          mkdir /home/kogito/.m2/repository
+        fi
+      workingDir: $(workspaces.source.path)
+    - name: compile-quarkus-native-binary
+      image: quay.io/kiegroup/kogito-builder:1.7
```

Comment:
@ruromero  why are we using the kogito-builder?

Reply:
Related to the above comment